plugins/woocommerce/client/admin/package.json

```diff
@@ -120,8 +120,8 @@
 		"@testing-library/react": "12.1.3",
 		"@testing-library/react-hooks": "7.0.2",
 		"@testing-library/user-event": "13.5.0",
-		"react-test-renderer": "17.0.2",
 		"@types/cookie": "^0.4.1",
+		"@types/debug": "4.1.12",
```

Comment:
I only added `@types/debug` to the dependencies because it was missing. The other changes were made by pnpm when I ran `pnpm add --save-dev @types/debug`.